frontend/views/GameView.tsx

```diff
@@ -29,7 +27,7 @@ export class GameView extends React.Component<GameViewProps> {
 						<PageTitle>{this.props.game.title}</PageTitle>
 						<PageSubtitle>{this.props.series.name}</PageSubtitle>
 					</PageHeader>
-					<GameBox image={this.props.image} environments={this.props.environments} />
+					<SmartGameBox imageId={this.props.game.imageId} gameId={this.props.game.id} />
```

Comment:
You don't need to pass `imageId` here because it already gets `gameId`, and it can get the imageId later from the game (for now it's okay for `SmartGameBox` to fetch the game again on its own).